android/build.gradle

```diff
@@ -55,7 +55,7 @@ android {
         targetCompatibility 1.8
     }
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion safeExtGet('minSdkVersion', 16)
```

Comment:
@apyousef This is the only interesting thing here.  I believe what's happening here is that RN 0.64+ has a higher minSdkVersion than 16, so we fail when linking against it.